config/jacoco.gradle

```diff
@@ -22,27 +29,38 @@ task jacocoTestReport(type: JacocoReport, dependsOn: "testStagingDebugUnitTest")
                          'android/**/*.*']
 
     def packagesExcluded = ['com/nimbl3/di/**',
+                            'com/nimbl3/ui/**/di/**',
                             'com/nimbl3/extension/**',
                             'com/nimbl3/**/extension/**']
-    // def filesExcluded = [ ]
-    // def fileFilter = fileGenerated + packagesExcluded + filesExcluded
+
     def fileFilter = fileGenerated + packagesExcluded
 
-    classDirectories = fileTree(
-        dir: "$buildDir/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
+    classDirectories.from = fileTree(
+        dir: "$project.rootDir/app/build/intermediates/javac/stagingDebug/compileStagingDebugJavaWithJavac/classes",
```

Comment:
Could you please check is this a correct path for java files? I think this path will be changed to `$buildDir/intermediates/javac/stagingDebug/classes` after upgraded to 0.8.5 https://github.com/redplanethotels/rphl-android-app/pull/2109